fsspec/implementations/dbfs.py

```diff
@@ -212,9 +212,9 @@ def mv(
             )
         except DatabricksException as e:
             if e.error_code == "RESOURCE_DOES_NOT_EXIST":
-                raise FileNotFoundError(e.message)
+                raise FileNotFoundError(e.message) from e
             elif e.error_code == "RESOURCE_ALREADY_EXISTS":
-                raise FileExistsError(e.message)
+                raise FileExistsError(e.message) from e
```

Comment:
I wonder if there is a level of verbosity in the upstream exception when we shouldn't do this. We already extracted the salient information from `e`.

Reply:
We don't get the traceback from e